### ISSUE TITLE
add two output maps in diagonal BiLSTM

### ISSUE THREAD

**Issue by powder21** [carpedm20/pixel-rnn-tensorflow#16]
`output_state_bw_with_last_zeros = tf.concat(1, [output_state_bw_except_last, dummy_zeros])`

should be

`output_state_bw_with_last_zeros = tf.concat(1, [dummy_zeros, output_state_bw_except_last])`

**Comment by ChengBinJin** [comment]
yes, I think you are right! In https://github.com/igul222/pixel_rnn theano code just do like that.

**Comment by lujiarui-iie** [comment]
yesyes, I think so!!!
